Docs/migration-guide-5.x-6.x.md

```diff
@@ -0,0 +1,38 @@
+## 5.x to 6.x Migration Guide
+
+- The Voice SDK has been updated for better Swift interoperability. The `TVO` class prefix has been removed from all Twilio Voice types.
+ 
+ ```swift
+    let connectOptions = ConnectOptions(accessToken: accessToken) { builder in
+        // build connect options using builder
+    }
+    let call = TwilioVoice.connect(options: connectOptions, delegate: self)
+ ```
+ 
+- This release has improved API for CallKit integration. In order to use CallKit with SDK, you must set `ConnectOptions.uuid` or `AcceptOptions.uuid` while making or answering a Call. When `ConnectOptions.uuid` or `AcceptOptions.uuid` is set, it is your responsibility to enable and disable the audio device. You should enable the audio device in `[CXProviderDelegate provider:didActivateAudioSession:]`, and disable the audio device in `[CXProviderDelegate provider:didDeactivateAudioSession:]`.
+
+```swift
+    func provider(_ provider: CXProvider, didActivate audioSession: AVAudioSession) {
+        audioDevice.isEnabled = true
+    }
+
+    func provider(_ provider: CXProvider, didDeactivate audioSession: AVAudioSession) {
+        audioDevice.isEnabled = false
+    }
+    
+    func providerDidReset(_ provider: CXProvider) {
+        audioDevice.isEnabled = false
+    }
+```
+
+Please note, if you are not using CallKit in your app, you must not set `ConnectOptions.uuid` or `AcceptOptions.uuid` while making or answering a call. The Voice SDK will enable the audio device for you when the `uuid` is `nil`. 
```

Comment:
Perhaps this paragraph can be omitted since the use of UUID is described in the previous paragraph. Maybe a few lines in the ConnectOptions sample code?

Reply:
I think it's OK to explicitly this call out.